File: java/facade/scr/main/java/fasade/videoUtils/Video.java
package fasade.videoUtils;

import java.util.ArrayList;
import java.util.List;

public class Video {
    private List<String> tasks;

    public Video(String sourceVideo) {
        this.tasks = new ArrayList<>();
        this.tasks.add(sourceVideo);
    }

    public List<String> getTasks() {
        return tasks;
    }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append(tasks.get(0)).append(" : {\n");
        for (String t : tasks) {
            if (!tasks.get(0).equals(t)) {
                sb.append(t).append("\n");
            }
        }
        sb.append('}');
        return sb.toString();
    }
}

File: python/facade/videoUtils/Video.py
class Video:
    def __init__(self, sourceVideo: str) -> None:
        self.tasks = []
        self.tasks.append(sourceVideo)

    def getTasks(self) -> list[str]:
        return self.tasks

    def toString(self) -> str:
        result = ""
        if len(self.tasks) < 1:
            return result

        result += self.tasks[0]

        for element in self.tasks:
            result += element
            result += "\n"

        return result